ddsenabler/examples/action/CMakeLists.txt

```diff
@@ -46,5 +46,17 @@ foreach(YAML_FILE_COMPLETE_PATH ${YAML_FILES})
         ${CMAKE_CURRENT_BINARY_DIR}/${YAML_FILE}.yml
         COPYONLY)
     install(FILES ${YAML_FILE_COMPLETE_PATH}
-        DESTINATION ${DATA_INSTALL_DIR}/ddsenabler/examples/ddsenabler/${BIN_INSTALL_DIR})
+        DESTINATION bin)
 endforeach()
+
+# Install persistence directory
+install(
+    DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}/../persistence
+    DESTINATION bin
+)
+
+# Install goals directory
+install(
+    DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}/goals
+    DESTINATION bin
```

Comment:
Please keep the examples separated in the bin folder, I believe they should be under a examples folder, and each of them in a specific subfolder.